include/envoy/stats/refcount_ptr.h

```diff
@@ -25,6 +25,8 @@ namespace Stats {
 // API in years.
 template <class T> class RefcountPtr {
 public:
+  using element_type = T;
```

Comment:
doc this? And also consider spelling it ElementType per Envoy style?

Reply:
This definition is taken from std::shared_ptr. I'll doc it.